configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,33 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    service_hash = [{
+      :directory_ref => settings[:bindir_id],
+      :id => "MCOService",
+      :guid => "7601FCEA-90B3-CC69-6A69-4087FBC7292D",
+      :win64 => settings[:win64],
+      :file => {
+        :id => "RubyWExe",
+        :source => settings[:rubyw],
+      },
+      # This service is installed with start set to demand because
+      # it won't be correctly configured to start right away. The
+      # puppet run that configures mcollective will allow it to start
+      # and set it to automatic
+      :serviceinstall => {
+        :id => "MCOServiceInstaller",
+        :description => "Puppet Labs server orchestration framework",
+        :displayname => "Marionette Collective Server",
```

Comment:
https://github.com/puppetlabs/puppet_for_the_win/blob/5e59d0e713f199253f86419ed17ca057fb9a7304/wix/puppet.wxs#L229 I don't think I have the authority to change the name